### PR TITLE
added action tests for search input and set search keyword

### PR DESCRIPTION
**Pull request by srhboo**

### CODE REVIEW

**Comment by auredufour** [comment]
👍

**Comment by auredufour** [comment]
bitmoji

File: src/redux/actions/index.js [hunk 1]
```diff
@@ -15,7 +10,3 @@ export const setSearchKeyword = value => ({
   payload: value
```
Comment (cgolinski) [inline src/redux/actions/index.js]: Is this a duplicate of line 5?

**Comment by turquoisemelon** [comment]
I can't see the data fetching actions in this version of the actions/index.js file. Is it possible this was done before the data fetching gets merged to master? @srhboo

**Comment by auredufour** [comment]
@turquoisemelon I think the PR was created before the fetch, it's why Github is warning that the branch has conflicts.

**Comment by turquoisemelon** [comment]
Then, I think the PR should be updated with the new master before it gets merged

**Comment by turquoisemelon** [comment]
Oh @srhboo already did that. Great!

**Pull request merged**